test/instrumentation/stability-utils.sh

```diff
@@ -29,37 +29,45 @@ stability_check_setup() {
   kube::golang::setup_env
 }
 
-find_files_to_check() {
-  find . -not \( \
-      \( \
-        -wholename './output' \
-        -o -wholename './_output' \
-        -o -wholename './_gopath' \
-        -o -wholename './release' \
-        -o -wholename './target' \
-        -o -wholename '*/third_party/*' \
-        -o -wholename '*/vendor/*' \
-        -o -wholename '*/hack/*' \
-        -o -wholename '*_test.go' \
-        -o -wholename './test/instrumentation/*.go' \
-        \) -prune \
-    \) \
-    \( -wholename '**/*.go' \
-  \)
+function find_files_to_check() {
+    # Similar to find but faster and easier to understand.  We want to include
+    # modified and untracked files because this might be running against code
+    # which is not tracked by git yet.
+    git ls-files -cmo --exclude-standard \
+        ':!:vendor/*' \
+        ':!:third_party/*' \
+        ':!:hack/*' \
+        ':!:*/testdata/*' \
```

Comment:
ditto

Reply:
it seems that testdata files are also referred in this script
```
 git ls-files -cmo  --exclude-standard | grep testdata | grep \.go$
staging/src/k8s.io/kubectl/pkg/cmd/edit/testdata/record.go
test/conformance/testdata/embed.go
test/instrumentation/testdata/pkg/kubelet/metrics/metrics.go
test/instrumentation/testdata/staging/src/k8s.io/metrics/metrics.go
test/typecheck/testdata/bad/bad.go
test/typecheck/testdata/good/_ignore/bad.go
test/typecheck/testdata/good/good.go
test/typecheck/testdata/good/testdata/bad.go

```

Reply:
ACK.  I must have cut-pasted this block and forgotten to remove it or something, thanks.